src/routes/visit.routes.js

```diff
@@ -37,11 +37,12 @@ visitRoutes.get('/visit/:id', async (req, res, next) => {
   }
 })
 
+// Clarify: why need to fetch non-existent visit on POST?
 visitRoutes.post('/visit', async (req, res, next) => {
   try {
-    const { patient_id } = req.body
-    const targetPatient = await patients.findOne({ where: { id: patient_id }})
-    if(targetPatient === null) throw new Error('No Patient Found')
+    // const { patient_id } = req.body
+    // const targetPatient = await patients.findOne({ where: { id: patient_id }})
+    // if(targetPatient === null) throw new Error('No Patient Found')
```

Comment:
Visits shouldn't exist without patients, that's why we have to make sure the patients are valid prior to creating a visit.